stacks-signer/src/signerdb.rs

```diff
@@ -421,9 +429,19 @@ static SCHEMA_3: &[&str] = &[
     "INSERT INTO db_config (version) VALUES (3);",
 ];
 
+static SCHEMA_4: &[&str] = &[
```

Comment:
I think you can make this a single schema now! I know it was merged in from another PR but that PR has yet to hit develop so why don't we combine these?

Reply:
Yeah, good point. I'll make that change.

Reply:
Fixed in ecae254ef.